flink-yarn/src/test/java/org/apache/flink/yarn/YarnApplicationFileUploaderTest.java

```diff
@@ -143,6 +165,22 @@ private static Map<String, String> getLibJars() {
         return libJars;
     }
 
+    private static Map<String, String> getFilesWithParentDir() {
+        final Map<String, String> filesWithParentDir = new HashMap<>(2);
+        final String xmlContent = "XML Content";
+
+        filesWithParentDir.put("conf/hive-site.xml", xmlContent);
+        filesWithParentDir.put("conf/ivysettings.xml", xmlContent);
+
+        return filesWithParentDir;
+    }
+
+    private static List<String> getExpectedClassPathWithParentDir() {
```

Comment:
nit: Currently this is used only by one test, why not just inline it?

Reply:
It can be rewritten as:
```
List<String> expectedClassPathWithParentDir = Arrays.asList("conf");
```

Reply:
Done.